test/test_transforms.py

```diff
@@ -1667,6 +1667,91 @@ def test_random_grayscale(self):
         # Checking if RandomGrayscale can be printed as string
         trans3.__repr__()
 
+    def test_random_erasing(self):
```

Comment:
@tejank10 I think there was a problem while resolving the conflict with master

Reply:
Okay, I'll figure it out.